### DISCUSSION TITLE
Slider Bar as a percentage

### DISCUSSION THREAD

**Discussion by cloudbr34k84** [Nerwyn/custom-card-features#63] [General]
How would i go about showing the slider as a percentage??
so if the battery was 100% then the slider bar would be full?

type: tile
entity: binary_sensor.beaus_bedroom_ensuite_vanity_leak_detector_top_shelf_water_leak
name: " Top Shelf"
features:
  - type: custom:service-call
    entries:
      - type: slider
        entity_id: sensor.beaus_bedroom_ensuite_vanity_leak_detector_top_shelf_battery
        range:
          - 0
          - 100
        tap_action:
          action: perform-action
          target:
            entity_id: >-
              binary_sensor.beaus_bedroom_ensuite_vanity_leak_detector_top_shelf_water_leak
          data: {}
        step: 1
        thumb: flat
        styles: |-
          :host {
            flex-flow: row;
          }
        label: Battery
        icon: mdi:battery
card_mod:
  style:
    ha-tile-info$: |
      .secondary:after {
        visibility: visible;
        content: " ⸱ {{ states('sensor.beaus_bedroom_ensuite_vanity_leak_detector_top_shelf_battery') }}%";
      }

**Comment by Nerwyn** [comment]
When I said your issue should've gone on discussions, I meant for future reference. I gave you [an answer there](https://github.com/Nerwyn/service-call-tile-feature/issues/62#issuecomment-2492648119).

**Reply by cloudbr34k84** [reply]
are you referring to the extra code

  :host {
    flex-flow: row;
  }
  .icon {
    padding: 11px;
  }
  .label {
    padding-right: 42px;
  }

**Reply by Nerwyn** [reply]
No, the entire config I provided. Setting the label to `'{{ value }}{{ unit }}'` and units to `%` will  show the percentage on the slider, and setting action to none will prevent you from changing it manually. Those styles just help the icon and label align better.
